web/src/beta/lib/core/engines/Cesium/useOverrideGlobeShader.ts

```diff
@@ -88,15 +91,61 @@ export const useOverrideGlobeShader = ({
 
     let replacedGlobeFS;
     try {
+      // TODO: Fix this befor merge
+      // replacedGlobeFS = new StringMatcher()
+      //   .replace(
+      //     [
+      //       "float diffuseIntensity = clamp(czm_getLambertDiffuse(czm_lightDirectionEC, normalize(v_normalEC)) * u_lambertDiffuseMultiplier + u_vertexShadowDarkness, 0.0, 1.0);",
+      //       "vec4 finalColor = vec4(color.rgb * czm_lightColor * diffuseIntensity, color.a);",
+      //     ],
+      //     "vec4 finalColor = reearth_computeImageBasedLightingColor(color);",
+      //   )
+      //   .execute(GlobeFS);
+      console.log("Shader execution start!!");
```

Comment:
This line should be executed when `heatmap` is true.
```
if(heatmap) {
  console.log("Shader execution start!!");
  // ...Some logic
  console.log("shader execution ended !!");
}
```

Reply:
You need to pass `heatmap` from [here](https://github.com/reearth/reearth/blob/9b9d4bcf792bc7bc374ff32fb66b7ce2c2671219/web/src/beta/lib/core/engines/Cesium/hooks.ts#L467) like below.
```
useOverrideGlobeShader({
  // ...
  heatmap: property?.terrain?.heatmap,
});
```